### DISCUSSION TITLE
Добавлять в буфер ссылку при клике на якорь

### DISCUSSION THREAD

**Discussion by furtivite** [doka-guide/platform#900] [Идеи]
Иногда возникает необходимость поделиться не статьёй, а конкретным параграфом. Я кликаю на иконку с якорем 

<img width="998" alt="Screenshot 2022-02-10 at 16 38 03" src="https://user-images.githubusercontent.com/50330458/153419352-77679667-354f-4d3c-aaba-1612ff66522e.png">

Чтобы поделиться статьёй дальше мне необходимо пойти в браузер, в окне копировать адрес и уже тогда отправлять. Кажется, это очень много лишних действий. 

Предлагается, добавлять данные по клику на якорную ссылку сразу в буфер обмена, чтобы можно было поделиться частью текста проще.

**Comment by solarrust** [comment]
Хорошая идея, но нужно показывать как-то визуально, что в буфер скопирована ссылка. Потому что это не нативное поведение.

**Reply by furtivite** [reply]
Может какой-то `title` или при наведении на него подсказку давать, с надписью «Скопировано в буфер»?

**Reply by solarrust** [reply]
Я бы пошла по пути сбора референсов и выбора из них подходящего направления.

**Comment by pepelsbey** [comment]
Можно сделать [как у нас на Веб-стандартах](https://web-standards.ru/articles/a11y-audit-basics/#section-3)

<img width="515" alt="image" src="https://user-images.githubusercontent.com/105274/153421544-d2c92584-f371-4933-8289-214d2cf992f6.png">

<img width="489" alt="image" src="https://user-images.githubusercontent.com/105274/153421592-b26fa239-5c65-4137-a560-ee6fc022d6d5.png">

**Reply by furtivite** [reply]
Аналогично сделано на пикабу https://youtu.be/7zTEect_rGM - вариант визуально понятный. Кажется.

**Comment by monochromer** [comment]
Ради экономии одного клика?

![image](https://user-images.githubusercontent.com/6412192/153421455-37f05972-2eb1-41d3-95da-2379450d609f.png)

**Reply by furtivite** [reply]
А если мобильный или планшет?

**Reply by monochromer** [reply]
longtap, но надо css поправить
